.gitignore

```diff
@@ -8,4 +8,5 @@ build
 dist
 minard.egg-info
 minard/static/docs
-settings.conf
\ No newline at end of file
+settings.conf
+minardenv.sh
```

Comment:
Can you remove this from the .gitignore.

Reply:
removed :)

Reply:
Apologies, I meant just remove minardenv.sh so that the .gitignore file is not changed.